samples/bluetooth/central_bas/prj.conf

```diff
@@ -17,6 +17,7 @@ CONFIG_BT_GATT_BAS_C=y
 CONFIG_BT_SCAN=y
 CONFIG_BT_SCAN_FILTER_ENABLE=y
 CONFIG_BT_SCAN_UUID_CNT=1
+CONFIG_BT_PRIVACY=y
```

Comment:
Is enabling this option required to get the directed advertising events?

Reply:
Well, as far as I remember central devices get Non-Resolvable Private address from Zephyr BLE stack. With this address type, directed advertising is impossible. To make it compatible I need to use either fixed address using this Kconfig option:

CONFIG_BT_SCAN_WITH_IDENTITY

or Resolvable Private Address using this Kconfig option:

CONFIG_BT_PRIVACY

Please correct me @joerchan if this is no longer true.

Reply:
You have to enable either `CONFIG_BT_SCAN_WITH_IDENTITY` or `CONFIG_BT_PRIVACY` to receive directed adv packet. `CONFIG_BT_PRIVACY` is recommended option.

Reply:
Thanks for clarifying.